micro_dl/utils/masks.py

```diff
@@ -94,3 +95,65 @@ def create_unimodal_mask(input_image, str_elem_size=3):
     thr_image = binary_opening(input_image > thr, str_elem)
     mask = binary_erosion(thr_image, str_elem)
     return mask
+
+
+def get_unet_border_weight_map(annotation, w0=10, sigma=5):
+    """
+    Generate the weight map for borders as specified in the UNet paper for an annotation.
+    Parameters
+    ----------
+    annotation: array-like
+        A 2D array of shape (image_height, image_width) contains annotation with each class labeled as an integer.
+
+    Returns
+    -------
+    array-like
+        A 2D array of shape (image_height, image_width)
+
+    TODO: Calculate boundaries directly and calculate distance from boundary of cells to another
+    TODO: The below method only works for UNet Segmentation only
+    """
+
+    assert annotation.dtype == np.uint8, "datatype expected uint8, it is {}".format(annotation.dtype)
```

Comment:
Please break the line at 80 characters, in this case the error message can go on a separate line